.github/workflows/publishDemo.yml

```diff
@@ -2,7 +2,7 @@ name: Deploy Demo
 
 on:
   pull_request:
-    branches: [ main ]
+    branches: [ master ]
```

Comment:
Why was this changed?

Reply:
This was changed because button uses `master` not `main`